code/modules/surgery/organs/eyes.dm

```diff
@@ -94,16 +94,23 @@
 		damaged = TRUE
 		if((organ_flags & ORGAN_FAILING))
 			eye_owner.become_blind(EYE_DAMAGE)
-		else if(damage > 30)
-			eye_owner.overlay_fullscreen("eye_damage", /atom/movable/screen/fullscreen/impaired, 2)
-		else
-			eye_owner.overlay_fullscreen("eye_damage", /atom/movable/screen/fullscreen/impaired, 1)
+			return
+
+		var/obj/item/clothing/glasses/eyewear = eye_owner.glasses
+		var/has_prescription_glasses = istype(eyewear) && eyewear.vision_correction
+
+		if(has_prescription_glasses)
```

Comment:
how are prescription glasses going to stop you from going blind because someone stabbed your eye with a screwdriver

Reply:
Stabbing someone in the eye gives them the nearsighted trait.  I've already argued in another PR about having books not be readable if a nearsighted person doesn't have glasses.  Why is that relevant?  Because eyestabs and any minor eye damage make you nearsighted by default.  I tried to argue that the quirk should have just be renamed to halfsighted since it makes more sense.

https://github.com/tgstation/tgstation/blob/72b640d1b496b1d76bf07dbb6f8b05ef8653e4e9/code/datums/elements/eyestab.dm#L97-L100